src/Components/Input.tsx

```diff
@@ -124,7 +124,7 @@ class Input extends React.Component<InputProps, InputState> {
       )
     }
 
-    return <StyledInput {...this.props} />
+    return <StyledInput {...this.props as any} />
```

Comment:
Why is this needed?

Reply:
I'm not too sure and now I feel bad for not investigating. One second...

Reply:
Yeah this is a weird one thats non-obvious unfortunately:

```
src/Components/Input.tsx(127,25): error TS2322: Type '{ children?: ReactNode; error?: boolean; block?: boolean; leftView?: Element; rightView?: Element...' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<any, ComponentState>> & Readonly<{ child...'.
  Type '{ children?: ReactNode; error?: boolean; block?: boolean; leftView?: Element; rightView?: Element...' is not assignable to type 'IntrinsicClassAttributes<Component<any, ComponentState>>'.
    Types of property 'ref' are incompatible.
      Type 'Ref<HTMLInputElement>' is not assignable to type 'Ref<Component<any, ComponentState>>'.
        Type '(instance: HTMLInputElement) => any' is not assignable to type 'Ref<Component<any, ComponentState>>'.
          Type '(instance: HTMLInputElement) => any' is not assignable to type '(instance: Component<any, ComponentState>) => any'.
            Types of parameters 'instance' and 'instance' are incompatible.
              Type 'Component<any, ComponentState>' is not assignable to type 'HTMLInputElement'.
                Property 'accept' is missing in type 'Component<any, ComponentState>'.

127     return <StyledInput {...this.props} />
```

I'm not sure why its saying that `accept` is missing as `styled-components`'s interface says that its optional, and `StyledInput` is just an instance of `styled.input`:

```ts
interface InputHTMLAttributes<T> extends HTMLAttributes<T> {
  accept?: string;
  ...
}
```

Reply:
Interesting! 🤔

Feel free to merge now if this is blocking you, otherwise I’ll take a look too see if I can understand it more when I get back home.

Reply:
No rush, but am definitely curious to know what's up; this only started appearing with the upgrade and _seems_ to be valid